src/classmentors/components/events/events.js

```diff
@@ -2112,7 +2111,7 @@ function ClmEventTableCtrl($scope, $q, $log, $mdDialog, $document,
         var parent = this;
 
         function loadEditor() {
-            var editor = ace.edit($document.querySelector('#editor'));
+            var editor = ace.edit(document.querySelector('#editor'));
```

Comment:
Separation of concern. The controller extends the scope by providing some values and event handlers. The controller doesn't need to a value comes from a textarea and ace editor.

And It easier to set a textarea as an editor in your template by adding a directive (the one we are using for singpath or something like [UI-Ace](https://github.com/angular-ui/ui-ace)) than to create that textearea in your template and then transform some arbitrary element from your controller.